systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -71,7 +78,7 @@ config_guest() {
 
 get_boot_params() {
   case $HYPERVISOR in
-     xen-pv|xen-domU)
```

Comment:
What if xen-domU is returned, removing the case would mean we no longer could process hypervisor where `xen-domU` would be returned.

Reply:
In that case, please remove/revert the change @dpassante